pages/common/pamdice.md

```diff
@@ -10,4 +10,4 @@
 
 - Make the produced pieces overlap by the specified amount horizontally and vertically:
 
-`pamdice {{[-o|-outstem]}} {{path/to/filename_stem}} {{[-he|-height]}} {{value}} {{[-w|-width]}} {{value}} {{[-ho|-hoverlap]}} {{value}} {{[-vo|-voverlap]}} {{value}} {{path/to/input.ppm}}`
+`pamdice {{[-o|-outstem]}} {{path/to/filename_stem}} {{[-he|-height]}} {{value}} {{[-w|-width]}} {{value}} {{[-h|-hoverlap]}} {{value}} {{[-v|-voverlap]}} {{value}} {{path/to/input.ppm}}`
```

Comment:
If I remember correctly, these overlap with `-help` and `-version`, thus -ho ans -vo are the shortest unique string
But I have not used the program so I don't know how it works in practice.

Reply:
I don't see any reason not to minimize the shortform to its shortest form. Consistency be damned if you can show to the user that the command could be shorter.

Reply:
The documentation uses `-` for longform. I made the changes to netbpm pages based purely on documentation and erring on the side of caution

I also managed to find a couple of errors in the documentation based on noticing inconsitencies and emailed the author about them.